### PR TITLE
Updated KiSAO

### PR DESCRIPTION
**Pull request by jonrkarr**
Addresses #1826 for KiSAO

Modified
- contact email
- home page
- domain
- description

Added
- Tags
- Browsers
- Releases
- Usage
- Publication
- Funding

### CODE REVIEW

File: ontology/kisao.md [hunk 1]
```diff
@@ -24,6 +29,48 @@ license:
...
+    title: Ontology Lookup Service
+    url: https://www.ebi.ac.uk/ols/ontologies/kisao
+  - label: BioPortal
+    title: BioPortal
+    url: https://bioportal.bioontology.org/ontologies/KISAO
+  - label: OntoBee
+    title: OntoBee
+    url: https://www.ontobee.org/ontology/KISAO  
+publications:
+  - id: https://pubmed.ncbi.nlm.nih.gov/22027554/
```
Comment (cthoyt) [inline ontology/kisao.md:69]:
```suggestion
  - id: https://www.ncbi.nlm.nih.gov/pubmed/22027554
```

File: ontology/kisao.md [hunk 2]
```diff
@@ -24,6 +29,48 @@ license:
...
+    type: annotation
+    description: BioSimulations is a repository of biosimulation projects.
+    examples:
+      - url: https://biosimulations.org/projects/Repressilator-Elowitz-Nature-2000
+        description: "Simulation of a synthetic oscillatory biochemical network"
+  - user: https://run.biosimulations.org/
+    type: annotation
+    description: runBioSimulations is a web application for execution biological simulations.
+    examples:
+      - url: https://run.biosimulations.org/runs?try=1
```
Comment (matentzn) [inline ontology/kisao.md:49]: Can you detail how KISAO ids can be witnessed on this website?

File: ontology/kisao.md [hunk 3]
```diff
@@ -24,6 +29,48 @@ license:
...
+        description: "Example simulation runs"
+  - user: https://biosimulators.org/
+    seeAlso: https://arxiv.org/abs/2203.06732
+    type: annotation
+    description: BioSimulators is a registry of biosimulation tools.
+    examples:
+      - url: https://biosimulators.org/simulators/tellurium
```
Comment (matentzn) [inline ontology/kisao.md:56]: How can I understand from here how KISAO is used?

File: ontology/kisao.md [hunk 4]
```diff
@@ -24,6 +29,48 @@ license:
...
+    type: annotation
+    description: The Simulation Experiment Description Markup Language (SED-ML) is a language for describing simulations and visualizations of their results.
+    examples:
+      - url: https://sed-ml.org/examples.html
+        description: "Several examples of simulations encoded in SED-ML"
+  - user: https://biosimulations.org/
+    type: annotation
+    description: BioSimulations is a repository of biosimulation projects.
+    examples:
+      - url: https://biosimulations.org/projects/Repressilator-Elowitz-Nature-2000
```
Comment (matentzn) [inline ontology/kisao.md:43]: How is KISAO used in this project? I cant see it form the page

**Review by matentzn — CHANGES_REQUESTED** [review]
Is it possible you misunderstood the `usages` field to be about projects you use, rather than projects that use KISAO? We require for ever usage a URL to resolve to a page that demonstrates the use of a concrete KISAO term.

File: ontology/kisao.md (hunk 3 above)
Comment (jonrkarr) [reply ontology/kisao.md:56]: These are projects that use KiSAO.

For example, BioSimulators uses KiSAO to categorize simulation tools (capture the kinds of models each tool can simulate). The specific KiSAO terms supported by each tool are presented in the "Algorithms" tab. For example, this displays the algorithms supported by the tellurium simulation tool: https://biosimulators.org/simulators/tellurium/latest#tab=algorithms.

File: ontology/kisao.md (hunk 4 above)
Comment (jonrkarr) [reply ontology/kisao.md:43]: BioSimulators uses KiSAO to keep track of the type of each simulation. The name of the KiSAO term is displayed on the landing tab for each simulation.

For the example above (https://biosimulations.org/projects/Repressilator-Elowitz-Nature-2000), the simulation type is CVODE (KISAO_0000019).

File: ontology/kisao.md (hunk 2 above)
Comment (jonrkarr) [reply ontology/kisao.md:49]: This application uses KiSAO to report which simulation algorithm was actually executed and with what parameter values. For various reasons, this is important to make transparent.

Names of KiSAO concepts are displayed 1 level deeper such as at https://run.biosimulations.org/runs/61fea483f499ccf25faafc4d#tab=log. This example shows that the simulation was executed using synchronous logical model simulation method (KISAO_0000449).

Comment (matentzn) [reply ontology/kisao.md:49]: Indeed, thank you for the clarification

File: ontology/kisao.md (hunk 4 above)
Comment (matentzn) [reply ontology/kisao.md:43]: Thanks

**Review by matentzn — APPROVED** [review]

**Review by matentzn — APPROVED** [review]

**Pull request merged**